kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -163,15 +161,15 @@ private PartitionedTableId getRecordTable(SinkRecord record) {
 
   /**
    * Create the table which doesn't exist in BigQuery for a (record's) topic when autoCreateTables config is set to true.
-   * @param record Kafka Sink Record to be streamed into BigQuery.
+   * @param topic Kafka Sink Record topic.
```

Comment:
nit: order of parameters doesn't match between with function

Reply:
Changed to order to align with function.